yt/python/yt/wrapper/tests/py3_only/test_typed_api.py

```diff
@@ -1709,3 +1709,49 @@ class RowWithBadNestedPostInit:
 
         with pytest.raises(YtError, match="bad post init: field2"):
             list(yt.read_table_structured(table, RowWithBadNestedPostInit))
+
+    @authors("ermolovd")
```

Comment:
Last time [I was renamed](https://github.com/ytsaurus/ytsaurus/commit/c75f2ded99947f25830f2828b022a05c12b26796#diff-86adac161190f2687305f8e81d0c3aa18b44d84725f36c9dc44dd0bbdc9ed6e2R1621) to `ermolovd`, so this time I just use his name.